apps/ledger-live-desktop/src/renderer/families/celo/LockFlowModal/steps/StepAmount.tsx

```diff
@@ -56,6 +56,7 @@ const StepAmount = ({
     <Box flow={1}>
       <SyncSkipUnderPriority priority={100} />
       {error && <ErrorBanner error={error} />}
+      {status.errors.sender && <ErrorBanner error={status.errors.sender} />}
```

Comment:
Do we want to be able to have both errors at the same time in the UI ?
What is the difference between `status` and the default `error` coming from the parent component ?